commands/help/topics/cpp.txt

```diff
@@ -1,4 +1,4 @@
 See these links/courses for learning C++:
 
-http://www.cplusplus.com/doc/tutorial/
-
+http://stackoverflow.com/a/388282
```

Comment:
Like other help topics, maybe we could add online resources instead of books, they are still useful but online ones should be added as well.